app/templates/public/speakers.hbs

```diff
@@ -4,7 +4,7 @@
       {{public/speaker-item speaker=speaker}}
     </div>
   {{/each}}
-  {{#infinity-loader infinityModel=model}}
+  {{#infinity-loader infinityModel=model triggerOffset=40 eventDebounce=30}}
```

Comment:
Increase it more, still only loads speakers when I am absolutely at the bottom

Reply:
Done